test/tree-sitter/testdata/php/StepDefinitions.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace LanguageService;
+
+class StepDefinitions 
+{
+    /**
+     * @Given ^a regexp$
```

Comment:
May I know why we are matching for `^` and `$` literally in `ExpressionBuilder.test.ts`?

Reply:
This is to make sure the entire regexp is extracted from code.